### PR TITLE
Simplify conflict of interest policy

### PR DESCRIPTION
**Pull request by ebullient**
- language for standard of compensation is "not excessive"
- add references to relevant Florida statute.
- Simplify determination of conflict and records, to avoid over-specification
- remove violation section, as already covered under bylaws (removing CFC/CF members.. )

### CODE REVIEW

File: policies/conflict-of-interest.md [hunk 1]
```diff
@@ -57,33 +58,21 @@ It is important to note that the existence of a conflict does not mean the trans
...
+The minutes of any CFC meeting that addresses conflict of interest disclosure or resolution will include:
+
+- The names of individuals with a potential conflict, the nature of their interest, and whether it was considered a conflict.
+- Details on alternative options reviewed, the CFC members who deliberated and voted, including how conflicts of interest were managed during these discussions.
+- The outcome of the vote, specifying the transaction's terms and approval date if relevant.
 
 #### Annual Disclosure and Compliance
 
-- CFC members (Councilors), EGC Members, and key employees (if any) must annually affirm that they have received, read, understood this policy, and that they agree to comply with it. They must disclose any financial interests and family relationships that could give rise to conflicts of interest.
+- CFC members (Councilors), EGC Members, and key employees (if any) must annually affirm that they have received, read, understood this policy, and that they agree to comply with it.
```
Comment (kenfinnigan) [inline policies/conflict-of-interest.md:73]: Does this mean the disclosing of conflicts of interest is covered by the policy itself, and does not need to be explicitly stated here?

File: policies/conflict-of-interest.md [hunk 2]
```diff
@@ -57,33 +58,21 @@ It is important to note that the existence of a conflict does not mean the trans
...
 - General CF members must affirm that they have received, read, understood this policy, and that they agree to comply with it, when applicable.
 
-## Violations
```
Comment (kenfinnigan) [inline policies/conflict-of-interest.md:81]: What is the process if a violation is found?

File: policies/conflict-of-interest.md (hunk 1 above)
Comment (ebullient) [reply policies/conflict-of-interest.md:73]: Yes. I think that's what it means.

File: policies/conflict-of-interest.md (hunk 2 above)
Comment (ebullient) [reply policies/conflict-of-interest.md:81]: It should be handled as we would any other violation (which is in the bylaws). We either keep that information in the bylaws (suspension/removal/disciplinary action), or we repeat it in each policy.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**